task/conftest-clair/0.1/conftest-clair.yaml

```diff
@@ -36,5 +36,32 @@ spec:
                    failures: (.failures // [])|map(.metadata.details.name) | unique
                  }' $(workspaces.conftest-ws.path)/clair-vulnerabilities.json || true)
         echo "${HACBS_TEST_OUTPUT:-${HACBS_ERROR_OUTPUT}}" | tee $(results.HACBS_TEST_OUTPUT.path)
-  workspaces:
-    - name: conftest-ws
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+  name: get-clair-scan
```

Comment:
For after the tasks are merged into a single task: I think it would probably be best to rename the resulting task to something like `clair-scan`